src/ConferenceList.jsx

```diff
@@ -1,13 +1,23 @@
 import React from "react";
 import ConferenceItem from "./ConferenceItem";
 import conferences from "./conferenceDataMock.json";
+import styled from "styled-components";
+
+const StyledUList = styled.ul`
```

Comment:
Rather than StyledUList, i'd call it something more declarative. Like StyledConferenceList. This will be clearer if you are looking through the DOM in the browser as to what is what

Reply:
We've also added this to the latest commit.